packages/editor/src/components/widgets/table/row/ReorderRow.tsx

```diff
@@ -37,22 +46,31 @@ export const ReorderRow = ({ id, updateOrder, children, ...props }: ReorderRowPr
       } else {
         console.log(`invalid drop item ${e.items[0]}`);
       }
+      if (!row.getIsSelected()) {
+        row.getToggleSelectedHandler()(e);
+      }
     }
   });
 
   return (
-    <MessageRowWithTr
-      {...dragProps}
-      {...dropProps}
-      ref={ref}
-      colSpan={2}
-      className={`dnd-row${isDragging ? ' dragging' : ''}${isDropTarget ? ' target' : ''}`}
-      {...props}
-    >
-      {children}
-      <td className='dnd-row-handle'>
-        <IvyIcon icon={IvyIcons.ChangeType} />
-      </td>
```

Comment:
I don't understand why this now needs to be inside of the cell... this Has nothing to do with the cell, you reorder the row not the cell!

Reply:
Nevermind maybe I could do it with an ReorderIcon-Wrapper as you suggested. 
![grafik](https://github.com/axonivy/inscription-client/assets/141223521/baadbe13-d4dd-471a-9175-b95b41211369)
![grafik](https://github.com/axonivy/inscription-client/assets/141223521/fd540c73-2535-4836-9260-55e32417b6fd)

What do you prefer?

Reply:
Both ways much better! I think I would go with the second one. But I woudn't call this just a ReorderIcon. Because if you do so, someone else could think he can just use this everywhere. Better something like ReorderWrapper or ReorderWrapperIcon...